### PR TITLE
Add transition to stats

### LINKED ISSUE #110: Add Transitions to stats
I want to add transition to stats that looks like counter starting from 0 to the current numeric values. Making it more aesthetic and visually appealing.

Screenshot 2024-10-17 130046

### PR DESCRIPTION
**Pull request by haseebzaki-07**
fixes #110 

This PR adds transition to the stats so that they appear like a counter in the testimonial section of the home page.

https://github.com/user-attachments/assets/4e6f8fac-230a-4be4-b6a1-4f9de7e40b90

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,4 +1,4 @@
 GEMINI_API_KEY=your_gemini_api_key
 MONGODB_URI=your_mongodb_connection_string
-NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=pk_test_a25vd2luZy1tb3RoLTU3LmNsZXJrLmFjY291bnRzLmRldiQ
```
Comment (sudhanshutech) [inline .env.example:3]: reason to replace the key?

**Review by sudhanshutech — APPROVED** [review]
I might not know why you are replacing the key from env because it might not be required and otherwise looks good to me

**Comment by himeshparashar** [comment]
@haseebzaki-07 the transition looks a bit slow make it fast

File: .env.example (hunk 1 above)
Comment (haseebzaki-07) [reply .env.example:3]: I was not able to start the frontend locally for which I Replaced it with my clerk generated key

**Comment by haseebzaki-07** [comment]
hey @himeshparashar @sudhanshutech I fixed the env keys and the transition time

**Comment by sudhanshutech** [comment]
thanks @haseebzaki-07 . I think its good to go @himeshparashar ✅.

**Pull request merged**